app/config/overrides/cabin.config.json

```diff
@@ -49,5 +49,14 @@
 	    "red": "./js/images/puck/thumb-red.png",
 	    "orange": "./js/images/puck/thumb-orange.png"
 	  }
+	},
+	"map": {
+		"additionalStats": [
+			{
+				"text": "{0} taking bathroom break",
```

Comment:
Maybe these should also have a `display` key for convenience, in case a config manager wants to toggle things on and off without having to remove / readd stats